migration/v0.34/apply/11-prometheus-operator-crds.sh

```diff
@@ -6,40 +6,52 @@ ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
 source "${ROOT}/scripts/migration/lib.sh"
 
 run() {
+  local cluster="${2:-both}"
```

Comment:
Use the environment variable instead of an argument if you go with an environment variable.